de-lib/src/main/java/org/iplantc/de/tools/client/views/manage/EditToolViewImpl.java

```diff
@@ -45,46 +41,66 @@
     TextField name;
 
     @UiField
-    TextArea desc;
+    TextArea description;
 
+    @Path("container.image.name")
     @UiField
     TextField  imgName;
 
+    @Path("container.image.tag")
     @UiField
     TextField tag;
 
+    @Path("container.image.url")
     @UiField
     TextField url;
 
     @UiField
     TextField version;
 
+    @Ignore
     @UiField
     FieldLabel nameLbl;
 
+    @Ignore
     @UiField
     FieldLabel versionLbl;
 
+    @Ignore
     @UiField
     FieldLabel imgLbl;
 
+    @Ignore
     @UiField
     TextField cpu;
 
+    @Ignore
     @UiField
     TextField memory;
 
+    @Ignore
     @UiField
     TextField network;
 
+    @Ignore
     @UiField
     TextField time;
 
-    Hidden toolId;
+    @Path("container.entryPoint")
+    @UiField
+    TextField entryPoint;
+
+    Hidden id;
```

Comment:
I didn't notice this last time I reviewed.  At a glance I didn't see it used anywhere, is it needed?